pkgs/applications/audio/gwc/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv
+, fetchFromGitHub
+, autoreconfHook
+, pkgconfig
+, alsaLib
+, libpulseaudio
+, gtk2
+, hicolor-icon-theme
+, libsndfile
+, fftw
+, perl
+, xdg_utils
+, vorbis-tools
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gwc";
+  version = "0.22-04";
+
+  src = fetchFromGitHub {
+    owner = "AlisterH";
+    repo = pname;
+    rev = version;
+    sha256 = "0xvfra32dchnnyf9kj5s5xmqhln8jdrc9f0040hjr2dsb58y206p";
+  };
+
+  buildInputs = [
+    alsaLib
+    libpulseaudio
+    gtk2
+    hicolor-icon-theme
+    libsndfile
+    fftw
+    perl
+    xdg_utils
+    vorbis-tools
+  ];
+
+  nativeBuildInputs = [
+    autoreconfHook
+    pkgconfig
```

Comment:
In new expressions, `pkg-config` should be used instead of the `pkgconfig` alias.